test/redhat_sigstore.yaml

```diff
@@ -0,0 +1,3 @@
+docker:
+  registry.access.redhat.com:
+    sigstore: https://access.redhat.com/webassets/docker/content/sigstore
```

Comment:
@mtrmac this test is failing with...

```
# time="2017-08-18 08:17:39.405794652Z" level=debug msg="Using registries.d directory /etc/containers/registries.d for sigstore configuration" 
# time="2017-08-18 08:17:39.406351625Z" level=debug msg=" Using "default-docker" configuration" 
# time="2017-08-18 08:17:39.406385188Z" level=debug msg=" No signature storage configuration found for registry.access.redhat.com/rhel7-atomic:latest" 
```
Does cri-o use the 'docker' transport type or is it something else?

Reply:
Those debug lines are from the `docker` transport; it just isn’t finding the docker/$hostname entry. I can’t immediately see anything wrong with it; perhaps modify `signatureTopLevel` to dump the dictionary vs. the looked up keys.